acos_exporter.py

```diff
@@ -81,88 +91,149 @@ def getauth(host):
             logger.error("password not provided.")
 
         payload = {'Credentials': {'username': uname, 'password': pwd}}
-        auth = json.loads(
-            requests.post("https://{host}/axapi/v3/auth".format(host=host), json=payload, verify=False).content.decode(
-                'UTF-8'))
+        try:
+            auth = json.loads(requests.post("https://{host}/axapi/v3/auth".format(host=host), json=payload,
+                                            verify=False, timeout=5).content.decode('UTF-8'))
+        except requests.exceptions.Timeout:
+            logger.error("Connection to {host} timed out. (connect timeout=5 secs)".format(host=host))
+            return ''
+
         if 'authresponse' not in auth:
             logger.error("Host credentials are not correct")
             return ''
         return 'A10 ' + auth['authresponse']['signature']
 
 
+def get_stats(body, endpoint, host_ip, headers):
+    batch_endpoint = "/batch-get"
+    logger.info("Uri - " + endpoint + batch_endpoint)
```

Comment:
Logging of list of end points is also needed

Reply:
It is done initially itself after parsing query params.